src/components/vote-form.js

```diff
@@ -1,8 +1,55 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import CandidateForm from "./candidate-form";
+
 import styled from "styled-components";
+import axios from "axios";
+
+export default function VoteForm(props) {
+  const [candidateList, setCandidateList] = useState([]);
+  const { name, voteCount, rank, id } = candidateList;
+
+  useEffect(() => {
+    getCandidateList();
+  }, [candidateList]);
```

Comment:
```suggestion
  }, []);
```
이 배열안에 candidateList가 들어있으면 getCandidateList 함수가 계속해서 실행된답니다 😅
[참고링크](https://velog.io/@jepjap93/useEffect%EC%97%90-%EB%8C%80%ED%95%9C-%EC%A7%A7%EC%9D%80-%EA%B0%80%EC%9D%B4%EB%93%9C)